includes/Elements/FluentForm.php

```diff
@@ -77,14 +77,13 @@ public static function get_fluent_forms_list()
 
 		$options = array();
 
-		if (defined('FLUENTFORM')) {
-			global $wpdb;
+		if ( defined('FLUENTFORM') && function_exists( 'wpFluent' ) ) {
```

Comment:
These changes related to PCP?

Reply:
yes. they suggest not to use direct $wpdb.